pkg/cmd/taskrun/list_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package taskrun
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/jonboulle/clockwork"
+	"github.com/knative/pkg/apis"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/testutil"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/reconciler/v1alpha1/pipelinerun/resources"
+	"github.com/tektoncd/pipeline/test"
+	tb "github.com/tektoncd/pipeline/test/builder"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestListTaskRuns(t *testing.T) {
+	now := time.Now()
+	trs := taskRuns(now)
```

Comment:
IMHO, writing the tests in the same style as `pipeline` is a lot easy to reason with, where different scenarios are separated into different test and where each test has a
```
Setup
Action
Validation
``` 
e.g. here the implementation detail of  `taskRuns(now)`  has to be known to the caller to understand what tasks and taskruns it creates.

Reply:
I agreed on this :+1: , hampers the redability

Reply:
:heart_eyes_cat: